Description: relaxing unicode-width
 unicode-width 1.8 is currently available
Author: Ed Neville <ed-debian@s5h.net>
Forwarded: not-needed
Last-Update: 2020-12-31
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: getopts/Cargo.toml
===================================================================
--- getopts.orig/Cargo.toml	2020-12-16 19:35:55.314069635 +0000
+++ getopts/Cargo.toml	2020-12-16 19:36:09.150188961 +0000
@@ -32,7 +32,7 @@
 package = "rustc-std-workspace-std"
 
 [dependencies.unicode-width]
-version = "0.1.5"
+version = "0.1.8"
 [dev-dependencies.log]
 version = "0.4"
 
